modules/qna/src/translations/fr.json

```diff
@@ -4,7 +4,7 @@
   "confirmDelete": "Voulez-vous supprimer la question?",
   "context": {
     "canTypeToCreate": "Vous pouvez taper dans la barre de sélection pour créer de nouveaux contextes.",
-    "create": "Créer \"{query}\"",
+    "createQuery": "Créer le contexte",
```

Comment:
If you keep the {query} part, you can display the name of the context that will be created

Reply:
When I initially resolved the createQuery translation tag, it showed up exactly as "Create {query}" in the modal, instead of resolving the entered name, hence the proposed change.